extensions/amp-tiktok/0.1/test/test-amp-tiktok.js

```diff
@@ -99,18 +101,14 @@ describes.realWin(
       expect(iframe.getAttribute('src')).to.contain('fr-FR');
     });
 
-    it('resizes using the fallback mechanism when no messages are received', async () => {
+    it.skip('resizes using the fallback mechanism when no messages are received', async () => {
```

Comment:
If this is staying skipped, could we file a flaky-test or other tracking issue to un-skip and link to it here?